### PR TITLE
20.2.3 update

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
-
-Refer to the [MVVM Support - Bind to a Collection of Dock Panels](https://docs.devexpress.com/WPF/11386/#non-existing-dock-panels-advanced) documentation topic for more information on how to bind a ViewModel collection to the **DockLayoutManager**.
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T963699)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# WPF Dock Layout Manager - Bind the View Model Collection with LayoutAdapters
+
+You can use the LayoutAdapters to place dock panels to existing dock groups, but also to groups which are not created yet. In this case, you should create the target group before you populate it with panels.
```
Comment (albertov05) [inline README.md]:
```suggestion
You can use LayoutAdapters to place dock panels in existing dock groups, as well as groups that have not yet been created. In this case, you should create the target group before you populate it with panels.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
+
+![image](https://user-images.githubusercontent.com/12169834/174016482-6466bbd9-3dde-4776-95bd-ac1423a09875.png)
+
+Follow the steps below to use the LayoutAdapter in your application:
+
+1. Create a LayoutAdapter class that implements the **ILayoutAdapter** interface.
+2. Implement the **Resolve** method. The method returns the target layout group name for the specified **DockLayoutManager** and the panel's ViewModel. When you implement the **Resolve** method, you can perform any action before a target name is returned.
```
Comment (albertov05) [inline README.md]:
```suggestion
2. Implement the **Resolve** method. This method returns the target layout group name for the specified **DockLayoutManager** and the panel's ViewModel. When you implement the **Resolve** method, you can perform any action before the target name is returned.
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
+
+You can use the LayoutAdapters to place dock panels to existing dock groups, but also to groups which are not created yet. In this case, you should create the target group before you populate it with panels.
+
+![image](https://user-images.githubusercontent.com/12169834/174016482-6466bbd9-3dde-4776-95bd-ac1423a09875.png)
+
+Follow the steps below to use the LayoutAdapter in your application:
+
+1. Create a LayoutAdapter class that implements the **ILayoutAdapter** interface.
+2. Implement the **Resolve** method. The method returns the target layout group name for the specified **DockLayoutManager** and the panel's ViewModel. When you implement the **Resolve** method, you can perform any action before a target name is returned. 
+3. Assign a LayoutAdapter instance to the **MVVMHelper.LayoutAdapter** attached property on the [DockLayoutManager](https://docs.devexpress.com/wpf/DevExpress.Xpf.Docking.DockLayoutManager) to add the LayoutAdapter to DockLayoutManager.
```
Comment (albertov05) [inline README.md]: What do you mean by "on the DockLayoutManager"? Is "on" the correct preposition?

File: README.md [hunk 4]
```diff
@@ -1,21 +1,32 @@
...
+<!-- default file list end -->
+
+## Documentation
+
+- [MVVM Support - Bind to a Collection of Dock Panels](https://docs.devexpress.com/WPF/11386/#non-existing-dock-panels-advanced)
+
+## More Examples
+
+- [WPF Dock Layout Manager - Bind the View Model Collection with IMVVMDockingProperties](https://github.com/DevExpress-Examples/wpf-docklayoutmanager-bind-view-model-collection-with-IMVVMDockingProperties)
+- [WPF Dock Layout Manager - Populate a LayoutGroup with the ViewModels collection](https://github.com/DevExpress-Examples/wpf-docklayoutmanager-display-viewmodels-collection-in-layoutgroup)
```
Comment (albertov05) [inline README.md]:
```suggestion
- [WPF Dock Layout Manager - Populate a LayoutGroup with the ViewModels Collection](https://github.com/DevExpress-Examples/wpf-docklayoutmanager-display-viewmodels-collection-in-layoutgroup)
```

File: README.md (hunk 3 above)
Comment (friedcucumber) [reply README.md]:
```suggestion
3. Assign a LayoutAdapter instance to the [DockLayoutManager](https://docs.devexpress.com/wpf/DevExpress.Xpf.Docking.DockLayoutManager)'s **MVVMHelper.LayoutAdapter** attached property to add the LayoutAdapter to the DockLayoutManager.

```

**Pull request merged**